client/src/pages/Home/HomePage.tsx

```diff
@@ -0,0 +1,125 @@
+import { FormEventHandler, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import clsx from 'clsx';
+import { Box, Grid, Typography, TextField, Button } from '@material-ui/core';
+import { DatePicker } from '@mui/lab';
+import { ParseableDate } from '@mui/lab/internal/pickers/constants/prop-types';
+import { generateQueryString } from '../../helpers/queryStringHelpers';
+import useStyles from './useStyles';
+import { useSnackBar } from './../../context/useSnackbarContext';
+
+export default function HomePage(): JSX.Element {
```

Comment:
I was wondering why is there a form submission within this component and I realized that this is the home page haha. Please adjust the base of the PR to only show changes pertaining to this feature :)